sdk/mongocluster/arm-mongocluster/package.json

```diff
@@ -61,7 +66,7 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.6.2",
-    "@azure/core-lro": "^3.0.0",
+    "@azure/core-lro": "3.0.0",
```

Comment:
why we have this change

Reply:
tempory resolve

Reply:
@MaryGao Could you explain why we use fixed version here? this change is from codegen.

Reply:
this should be a bug, @kazrael2119 could you help fix the codegen?

Reply:
fixed in https://github.com/Azure/autorest.typescript/pull/2655/commits/c669e6a0288053097b844f6b07b712f860b3d1a5